test/utils/helpers.ts

```diff
@@ -36,3 +36,38 @@ export const getPoolData = async (testEnv: TestEnv) => {
 
   return poolData;
 };
+
+export const newGetUserData = async (
```

Comment:
The `new` keyword is not clear. Can't this function be merged with above `getPoolData` and `getUserData`? What is the difference? I think they are same.